README.md

```diff
@@ -91,9 +120,10 @@ Dataset().export(save_path='output.zip', local_archive_path='clarifai-data-proto
 ```
 
 
-### Interacting with Inputs
+### ⌨️ Interacting with Inputs
```

Comment:
```suggestion
##  💾 Interacting with Inputs
```
Inputs is also a data, so having keyboard icon this seems misleading wr.t python `input()`